README.md

```diff
@@ -314,14 +319,14 @@ To disable array parsing entirely, set `parseArrays` to `false`.
 
 ```javascript
 var noParsingArrays = qs.parse('a[]=b', { parseArrays: false });
-assert.deepEqual(noParsingArrays, { a: { '0': 'b' } });
+assert.deepEqual(noParsingArrays, { a: { 0: 'b' } });
```

Comment:
did you perhaps accidentally autoformat this file? if so please revert it :-)

Reply:
oh yea by accident